lib/array.rb

```diff
@@ -0,0 +1,16 @@
+class Array
+
+  def molliesinject(start_point = nil, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.